### PR TITLE
[docs] Add missing tvOS support details

### PR DESCRIPTION
**Pull request by reichhartd**
# Why

The docs previously did not show that several Expo libraries support tvOS.  
- **expo-network**: since v6.0.0 (SDK 51)  
- **expo-status-bar**: always supported (JS-only)  
- **expo-router**: since SDK 51 ([see building for TV](https://docs.expo.dev/guides/building-for-tv/#limitations))

# How

Update the docs to reflect tvOS support for the above libraries.

# Test Plan

/

# Checklist

- [ ] I added a `changelog.md` entry and rebuilt the package sources according to [this short guide](https://github.com/expo/expo/blob/main/CONTRIBUTING.md#-before-submitting)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).
- [X] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)

### CODE REVIEW

File: docs/pages/versions/unversioned/sdk/status-bar.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: A library that provides the same interface as the React Native Stat
 sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-status-bar'
 packageName: 'expo-status-bar'
 iconUrl: '/static/images/packages/expo-status-bar.png'
-platforms: ['android', 'ios', 'web']
+platforms: ['android', 'ios', 'web', 'tvos']
```
Comment (douglowder) [inline docs/pages/versions/unversioned/sdk/status-bar.mdx:8]: Status bar code will compile and run on tvOS, but no status bar will show. We might want to note that somewhere.

**Review by douglowder — APPROVED** [review]

**Comment by douglowder** [comment]
Thanks for catching this and submitting the changes!

File: docs/pages/versions/unversioned/sdk/status-bar.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/versions/unversioned/sdk/status-bar.mdx:8]: I think it's worth adding a callout on this page.

Comment (amandeepmittal) [reply docs/pages/versions/unversioned/sdk/status-bar.mdx:8]: I'll follow up on this in a separate PR.

**Review by amandeepmittal — APPROVED** [review]

**Pull request merged**